nwbwidgets/test/test_ecephys.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from pynwb import TimeSeries
+from nwbwidgets.view import default_neurodata_vis_spec
+from nwbwidgets.ecephys import show_lfp, show_spectrogram
+from pynwb.ecephys import LFP, ElectricalSeries
+from hdmf.common import DynamicTableRegion
+
+
+def test_show_lfp():
+    
```

Comment:
See the ecephys tutorial for how to properly set up the LFP object https://pynwb.readthedocs.io/en/stable/tutorials/domain/ecephys.html#sphx-glr-tutorials-domain-ecephys-py